app/views/static/index.html.erb

```diff
@@ -3,5 +3,5 @@
 <p>This is a splash page. It is amazing.</p>
 
 <% if user_signed_in? %>
-  curl -X GET --header 'Authorization: Bearer <%= JWTWrapper.encode({ user_id: current_user.id }) %>' '<%= search_url(q: 'whatever')%>'
+  curl -X GET --header 'Authorization: Bearer <%= JWTWrapper.encode({ user_id: current_user.id }) %>' '<%= api_v1_search_url(q: 'whatever')%>'
```

Comment:
Is there any reason to/not to variableize (not a word) the API version here rather than hard coding it in?

Reply:
This is totally a sample code thing we'll pull out once we make a real splash page and docs.